src/ui/components/WorkbookIcon/WorkbookIcon.scss

```diff
@@ -7,5 +7,6 @@
 
     &__letters {
         text-align: center;
+        color: #000;
```

Comment:
color: var(--g-color-text-primary); ?
in the dark theme should be different color

Reply:
no, it should be black in both themes.